internal/agent/device/lifecycle/manager.go

```diff
@@ -59,11 +66,14 @@ func NewManager(
 		deviceName:           deviceName,
 		enrollmentUIEndpoint: enrollmentUIEndpoint,
 		managementCertPath:   managementCertPath,
+		managementKeyPath:    managementKeyPath,
 		deviceReadWriter:     deviceReadWriter,
 		enrollmentClient:     enrollmentClient,
```

Comment:
there is also the managementClient I believe the certs are in its memory?

Reply:
I've looked around and it seems like it just has the path to the cert, not the cert itself, unless I missed it. 
https://github.com/flightctl/flightctl/blob/ebe6e9a425c7ee1f164f729df1b534808709617b/internal/agent/device/bootstrap.go#L265